lxqt-config/lxqt-config-input/lxqt-config-input_fr.ts

```diff
@@ -53,18 +53,19 @@
     <message>
         <location filename="../../../keyboardconfig.ui" line="122"/>
         <source>Repeat interval:</source>
-        <translation type="unfinished">Intervalle de répétition :</translation>
+        <translation>Délai de répétition :</translation>
     </message>
     <message>
         <location filename="../../../keyboardconfig.ui" line="129"/>
         <source>Type in the following box to test your keyboard settings</source>
-        <translation type="unfinished">Tapez dans le champ ci-dessous pour tester vos paramètres clavier</translation>
+        <translatorcomment>Tapez dans le champ ci-dessous pour tester vos paramètres clavier</translatorcomment>
+        <translation type="unfinished">Saisissez une phrase pour tester les réglages</translation>
```

Comment:
it comes by default from qt-linguist when i don't explicitly validate the translation.
when I am not 100% sure it's correct, I prefer leaving it as unfinished so someone else can still review it.